test/ConnectionTest.cpp

```diff
@@ -2,6 +2,7 @@
 #include <gtest/gtest.h>
 
 #include "DBConnection.hpp"
+#include "DBConnectionCaster.hpp"
```

Comment:
Seems redundant

Reply:
@malirod , removed

Reply:
What did you remove? This whole file should disappear from this pr.

Reply:
@malirod , seems I forgot about this `include` after `git rebase`. Now `#include "DBConnectionCaster.hpp"` exactly deleted